source/databases-collections.txt

```diff
@@ -84,18 +84,25 @@ documents and its type is consistent throughout the collection, |vsce|
 displays an icon indicating that field's data type.
 
 Indexes
-```````
+~~~~~~~
 
-Your collections's indexes are listed under the :guilabel:`Indexes` heading.
-When you expand an index, each index key appears with an icon designating
-its type. Index key types include:
+Your collections's indexes are listed under the :guilabel:`Indexes` 
+heading. When you expand an index, each index key appears with an icon 
+designating its type. Index key types include:
 
 - Ascending
 - Descending
 - Geospatial (2d, 2dsphere, geoHaystack)
 - Text
 - Hashed
 
+.. note::
+
+   You can open a :ref:`JavaScript Playground <vsce-crud>` 
```

Comment:
[nit] Can we say "MongoDB Playground" or just "Playground"? That's more in-line with the rest of the docs.